docs/introduction/overview.rst

```diff
@@ -2,27 +2,29 @@
 Before you start
 ################
 
-| This tutorial offers Java developers all the needed information to build a complete blockchain application on the Horizen Sidechain system.
+| This tutorial offers Java developers all the information needed to build a complete blockchain application on the Horizen Sidechain system.
 
 | Apart from Java competency, this tutorial assumes that the reader has a high-level understanding of how blockchain-based distributed software works.
 
-So, concepts such as Transactions, UTXO's, Blocks, Validation, Confirmation, Consensus, Unique chain, and chain forks, Hash Function, Private/Public key, and Signing should be known and understood, as well as the concept of a network of nodes and node communication.
+So, you should be comfortable with concepts such as Transactions, UTXO's, Blocks, Validation, Confirmation, Consensus, Unique Chain, Chain Forks, Hash Function, Private/Public key, and Signing, along with the concept of a network of nodes and node communication.
 
-If the above words are new to you, you can start by exploring the Horizen Academy website's material (`link <https://academy.horizen.global/>`_). Also, the original whitepaper by Satoshi Nakamoto, "Bitcoin: A Peer-to-Peer Electronic Cash System" (`link <https://bitcoin.org/bitcoin.pdf>`_), can be a good starting point. Direct experience with an existing blockchain software is also a very useful prerequisite. For that, you can install the Horizen "zend" software from (`Github <https://github.com/HorizenOfficial/zen>`_), and explore its rpc command interface and "regtest" mode.
+If the above words are new to you, you can start by exploring the Horizen Academy website's material (`link <https://academy.horizen.global/>`_). Also, the original whitepaper by Satoshi Nakamoto, "Bitcoin: A Peer-to-Peer Electronic Cash System" (`link <https://bitcoin.org/bitcoin.pdf>`_), can be a good starting point. Direct experience with an existing blockchain software is also useful. For that, you can install the Horizen "zend" software from (`Github <https://github.com/HorizenOfficial/zen>`_), and explore its RPC command interface and "regtest" mode.
```

Comment:
If the above terms are new to you, you can start by exploring the Horizen Academy website's material (`link <https://academy.horizen.global/>`_). Also, the original whitepaper by Satoshi Nakamoto, "Bitcoin: A Peer-to-Peer Electronic Cash System" (`link <https://bitcoin.org/bitcoin.pdf>`_), can be a good starting point. Direct experience with an existing blockchain software is also useful. For that, you can install the Horizen zend software from (`Github <https://github.com/HorizenOfficial/zen>`_), and explore its RPC command interface and "regtest" mode.